README.md

```diff
@@ -23,11 +23,30 @@
 # Fingerprint Pro Cloudflare worker
 The Cloudflare worker is responsible for delivering the latest fingerprinting client-side logic and proxying identification requests and responses between your site and Fingerprint Pro's APIs.
```

Comment:
Consdier aligning the introduction and overall structure with Cloudfront: https://github.com/fingerprintjs/fingerprint-pro-cloudfront-integration 
and Azure https://github.com/fingerprintjs/fingerprint-pro-azure-integration

Reply:
Updated description